### ISSUE TITLE
The accuracy of the model drastically dropped

### ISSUE THREAD

**Issue by ehsankf** [pytorch/pytorch#38192]
## 🐛 Bug
I am training a model on MNIST and during the training the accuracy of the model drops drastically to 10%.

Epoch: 37
468 469 Loss: 0.078 | Acc: 97.338% (58403/60000)
78 79 Loss: 0.046 | Acc: 98.860% (9886/10000)
Best Test Score tensor(98.8800)
Epoch:  38

Epoch: 38
468 469 Loss: 0.068 | Acc: 97.795% (58677/60000)
78 79 Loss: 0.053 | Acc: 98.780% (9878/10000)
Best Test Score tensor(98.8800)
Epoch:  39

Epoch: 39
468 469 Loss: 0.418 | Acc: 85.373% (51224/60000)
78 79 Loss: 2.294 | Acc: 12.560% (1356/10000)
Best Test Score tensor(98.8800)
Epoch:  40

Epoch: 40
468 469 Loss: 2.302 | Acc: 11.133% (6680/60000)
78 79 Loss: 2.301 | Acc: 10.610% (1061/10000)
Best Test Score tensor(98.8800)
=====> Saving checkpoint...
Epoch:  41

## To Reproduce

1.
1.
1.

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
Nvidia driver version: 418.67
cuDNN version: Could not collect

Versions of relevant libraries:
[pip] numpy==1.18.1
[pip] torch==1.4.0
[pip] torchfile==0.1.0
[pip] torchvision==0.5.0
[conda] _pytorch_select           0.2                       gpu_0  
[conda] blas                      1.0                         mkl  
[conda] cudatoolkit               10.0.130                      0  
[conda] mkl                       2019.4                      243  
[conda] mkl-service               2.3.0            py37he904b0f_0  
[conda] mkl_fft                   1.0.15           py37ha843d7b_0  
[conda] mkl_random                1.1.0            py37hd6b4f25_0  
[conda] numpy                     1.18.1           py37h4f9e942_0  
[conda] numpy-base                1.18.1           py37hde5b4d6_1  
[conda] pytorch                   1.4.0           py3.7_cuda10.0.130_cudnn7.6.3_0    pytorch
[conda] torch                     1.2.0                    pypi_0    pypi
[conda] torchfile                 0.1.0                    pypi_0    pypi
[conda] torchvision               0.5.0                py37_cu100    pytorch

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by kshitij12345** [comment]
@ehsankf 
Hi,

I believe, this belongs to the discussion forum - http://discuss.pytorch.org

**Comment by ehsankf** [comment]
Are you sure it is not a bug in Pytorch?

**Comment by ehsankf** [comment]
> @ehsankf
> Hi,
> 
> I believe, this belongs to the discussion forum - http://discuss.pytorch.org

Are you sure it isn't a bug in Pytorch?

**Comment by malfet** [comment]
@ehsankf Is it always reproducible for you? Have you made any changes to the dataset?

**Comment by ptrblck** [comment]
Double post from the [forum](https://discuss.pytorch.org/t/accuracy-of-the-model-drastically-drops/80388/6?u=ptrblck).
My best suggestion is that the model is overtraining (epoch22 - epoch42 yield a noisy accuracy around 97.2%) and an "unlucky" update step might knock the model out.
Suggested workaround: lower the learning rate or try to use an adaptive optimizer.

**Comment by ehsankf** [comment]
I progressively lower the learning rate and it worked. I believe the network jumped out from minimum.
